app/controllers/performance_dashboards/overview_controller.rb

```diff
@@ -71,7 +73,9 @@ def download
     end
 
     private def set_key
-      @key = report_class.detail_method(params.dig(:filters, :key))
+      key_from_params = params.dig(:filters, :key).presence
+      # FIXME: this may crash because key can be missing from params
```

Comment:
Can we protect against failure by picking a default? I believe the initial page load would have the logic for that.

Reply:
My understanding is this is only used when rendering the details modal; at that point we should expect the key to be present, right? I guess my assumption is that this **should** be an error, it would be incorrect to use a default value. If that's not the case, can you help me understand how I could replicate this behavior?